replication-api-impl/src/test/java/com/connexta/replication/api/impl/worker/WorkerManagerTest.java

```diff
@@ -251,15 +249,14 @@ void testDestroy() {
 
   @Test
   void testNoLocalSiteConfigured() {
-    // setup
+    // given
     when(properties.getLocalSite()).thenReturn("");
-    when(siteManager.get(LOCAL_SITE)).thenThrow(NotFoundException.class);
+    when(siteManager.get("")).thenThrow(NotFoundException.class);
```

Comment:
Whoops, I think we can just delete the line right above this then put it back to the way it was and it should work.